DOCS.md

```diff
@@ -69,270 +70,325 @@ In a Sponsored Ad Break flow, once the user hits a mid-roll break with a true[X]
 
+You may initialize `TruexAdRenderer` early (a few seconds before the next pod even starts) in order to give it extra time to make the ad request. The renderer will output an [`onFetchAdComplete`](#onfetchadcomplete) event at completion of this ad request. This event can be used to facilitate the implementation of a timeout or loading indicator, and when to make the call to `start`.
 
-#### initWithUrl
+The parameters for this method call are:
 
-    - (id)initWithUrl:(NSString *)creativeUrl adParameters:(NSDictionary*)adParameters slotType:(NSString *)slotType optimizelyProjectId:(NSString*)optimizelyProjectId optimizelyUserId:(NSString*)optimizelyUserId
+* `creativeURL`: true[X] asset url returned by SSAI. In the example of Uplynk, this would correspond to `response.ads.breaks[0].ads[0].creative`
+* `adParameters`: AdParameters as returned by SSAI. In the example of Uplynk, this would correspond to `response.ads.breaks[0].ads[0].adParameters`
+* `slotType`: the type of the current ad pod, `PREROLL` or `MIDROLL`
 
-This method will be called by the app code in order to instantiate the `TruexAdRenderer`. The renderer will parse out the `creativeURL` and `adParameters` passed to it and make a request to the true[X] ad server to see what ads are available.
 
-You may instantiate `TruexAdRenderer` early (a few seconds before the next pod even starts) in order to give it extra time to make the ad request.
+#### `start`
 
-The parameters for this method call are:
+```objective-c
+    - (void)start:(UIViewController* _Null_unspecified)baseViewController;
+```
 
-* `creativeURL`: true[X] asset url returned by Uplynk. In the example response, this would correspond to `response.ads.breaks[0].ads[0].creative`
-* `adParameters`: AdParameters as returned by Uplynk. In the example response, this would correspond to `response.ads.breaks[0].ads[0].adParameters`
-* `podType`: the type of the current ad pod, `PREROLL` or `MIDROLL`
-* `optimizelyProjectId`: (OPTIONAL) only pass in an Optimizely Project ID if you intend to integrate the functionality of TruexAdRenderer with your existing Optimizely experimentation instance. Otherwise please pass in `nil`.
-* `optimizelyUserId`: (OPTIONAL) only pass in an Optimizely User ID if you intend to integrate the functionality of TruexAdRenderer with your existing Optimizely experimentation instance. Otherwise please pass in `nil`.
+This method should be called by the app code when the app is ready to display the true[X] unit to the user. This can be called anytime after the unit is initialized. 
 
+The app should have as much extraneous UI hidden as possible, including player controls, status bars and soft buttons/keyboards.
 
-#### start
+After calling `start`, the app code should wait for a [terminal event](#terminal-events) before taking any more action, while keeping track of whether or not the [`AD_FREE_POD`](#ad_free_pod) event has fired.
```

Comment:
Nit: perhaps we should standardize in this doc between using all caps for the event names versus the delegate format:
`AD_FREE_POD` vs `onAdFreePod`

Reply:
Yeah, I wanted to use the delegate format everywhere -- just missed a couple spots. Thanks for catching these.